network/alsp/manager/manager.go

```diff
@@ -79,6 +88,9 @@ type MisbehaviorReportManager struct {
 
 	// workerPool is the worker pool for handling the misbehavior reports in a thread-safe and non-blocking manner.
 	workerPool *worker.Pool[internal.ReportedMisbehaviorWork]
+
+	// DecayFunc is the function that calculates the decay of the spam record.
+	DecayFunc SpamRecordDecayFunc
```

Comment:
```suggestion
	decayFunc SpamRecordDecayFunc
```

see https://github.com/onflow/flow-go/pull/4485/files#r1259083046

Reply:
fixed in https://github.com/onflow/flow-go/pull/4485/files/26e950f74a10a36000dc982c43dd790a0891d38e#r1292248875

2e027bc40f1d064bf0a28ebc783153c5b97ea240